### PR TITLE
[istio][docs] Federation and multicluster requirements

### PR DESCRIPTION
**Pull request by apolovov**
## Description
Cluster requirements for Federation and Multicluster clarified.

## Why do we need it, and what problem does it solve?
Fixes https://github.com/deckhouse/deckhouse/issues/7841.

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [x] Documentation updated according to the changes.
- [ ] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: istio
type: chore
summary: Cluster requirements for Federation and Multicluster clarified.
impact: low
```

### CODE REVIEW

File: modules/110-istio/docs/README_RU.md [hunk 1]
```diff
@@ -214,6 +214,11 @@ Istio позволяет осуществлять сбор трейсов с п
 
 ### Федерация
 
+#### Требования к кластерам, вступающим в федерацию
```
Comment (elenashliaga) [inline modules/110-istio/docs/README_RU.md:217]:
```suggestion
#### Требования к кластерам
```

File: modules/110-istio/docs/README_RU.md [hunk 2]
```diff
@@ -214,6 +214,11 @@ Istio позволяет осуществлять сбор трейсов с п
...
+
+* У каждого кластера должен быть уникальный домен кластера (параметр [clusterDomain](../../installing/configuration.html#clusterconfiguration-clusterdomain) в интерфейсе `ClusterConfiguration`, по умолчанию `cluster.local`).
```
Comment (elenashliaga) [inline modules/110-istio/docs/README_RU.md:219]:
```suggestion
* У каждого кластера должен быть уникальный домен в параметре [`clusterDomain`](../../installing/configuration.html#clusterconfiguration-clusterdomain) в интерфейсе _ClusterConfiguration_, по умолчанию `cluster.local`.
```

File: modules/110-istio/docs/README_RU.md [hunk 3]
```diff
@@ -214,6 +214,11 @@ Istio позволяет осуществлять сбор трейсов с п
...
+* Подсеть подов и сервисов **не обязательно** должны быть уникальными (параметры [podSubnetCIDR](../../installing/configuration.html#clusterconfiguration-podsubnetcidr) и [serviceSubnetCIDR](../../installing/configuration.html#clusterconfiguration-servicesubnetcidr) интерфейса `ClusterConfiguration`).
```
Comment (elenashliaga) [inline modules/110-istio/docs/README_RU.md:220]:
```suggestion
* Адресные пространства подов и сервисов в параметрах [`podSubnetCIDR`](../../installing/configuration.html#clusterconfiguration-podsubnetcidr) и [`serviceSubnetCIDR`](../../installing/configuration.html#clusterconfiguration-servicesubnetcidr) интерфейса _ClusterConfiguration_  не обязаны быть уникальными.
```

File: modules/110-istio/docs/README_RU.md [hunk 4]
```diff
@@ -246,6 +251,11 @@ Istio позволяет осуществлять сбор трейсов с п
 
 ### Мультикластер
 
+#### Требования к кластерам, которые объединяются в мультикластер
```
Comment (elenashliaga) [inline modules/110-istio/docs/README_RU.md:254]:
```suggestion
#### Требования к кластерам
```

File: modules/110-istio/docs/README_RU.md [hunk 5]
```diff
@@ -246,6 +251,11 @@ Istio позволяет осуществлять сбор трейсов с п
...
+
+* Домены кластеров **должны** быть одинаковыми для всех членов мультикластера (параметр [clusterDomain](../../installing/configuration.html#clusterconfiguration-clusterdomain) интерфейса `ClusterConfiguration`, по умолчанию `cluster.local`).
```
Comment (elenashliaga) [inline modules/110-istio/docs/README_RU.md:256]:
```suggestion
* Домены кластеров должны быть одинаковыми для всех членов мультикластера (параметр [clusterDomain](../../installing/configuration.html#clusterconfiguration-clusterdomain) интерфейса `ClusterConfiguration`, по умолчанию `cluster.local`).
```

File: modules/110-istio/docs/README_RU.md [hunk 6]
```diff
@@ -246,6 +251,11 @@ Istio позволяет осуществлять сбор трейсов с п
...
+* Подсеть подов и сервисов **должны** быть уникальными для каждого члена мультикластера (параметры [podSubnetCIDR](../../installing/configuration.html#clusterconfiguration-podsubnetcidr) и [serviceSubnetCIDR](../../installing/configuration.html#clusterconfiguration-servicesubnetcidr) интерфейса `ClusterConfiguration`).
```
Comment (elenashliaga) [inline modules/110-istio/docs/README_RU.md:257]:
```suggestion
* Адресные пространства подов и сервисов в параметрах [`podSubnetCIDR`](../../installing/configuration.html#clusterconfiguration-podsubnetcidr) и [`serviceSubnetCIDR`](../../installing/configuration.html#clusterconfiguration-servicesubnetcidr) интерфейса _ClusterConfiguration_ должны быть уникальными для каждого члена мультикластера.
```

File: modules/110-istio/docs/README.md [hunk 7]
```diff
@@ -210,6 +210,11 @@ Below are their fundamental differences:
 
 ### Federation
 
+#### Requirements for clusters joining the federation
```
Comment (elenashliaga) [inline modules/110-istio/docs/README.md:213]:
```suggestion
#### Requirements for clusters
```

File: modules/110-istio/docs/README.md [hunk 8]
```diff
@@ -210,6 +210,11 @@ Below are their fundamental differences:
...
+
+* Each cluster must have a unique cluster domain ([clusterDomain](../../installing/configuration.html#clusterconfiguration-clusterdomain) setting in `ClusterConfiguration` interface).
```
Comment (elenashliaga) [inline modules/110-istio/docs/README.md:215]:
```suggestion
* Each cluster must have a unique domain in the [`clusterDomain`](../../installing/configuration.html#clusterconfiguration-clusterdomain) parameter in the _ClusterConfiguration_ interface, the default value is `cluster.local`.
```

File: modules/110-istio/docs/README.md [hunk 9]
```diff
@@ -243,6 +248,11 @@ To establish a federation, you must:
 
 ### Multicluster
 
+#### Requirements for clusters that are combined into a multicluster
```
Comment (elenashliaga) [inline modules/110-istio/docs/README.md:251]:
```suggestion
#### Requirements for clusters
```

File: modules/110-istio/docs/README.md [hunk 10]
```diff
@@ -243,6 +248,11 @@ To establish a federation, you must:
...
+
+* Cluster domains **should** be the same for all members of multicluster ([clusterDomain](../../installing/configuration.html#clusterconfiguration-clusterdomain) setting in `ClusterConfiguration` interface).
```
Comment (elenashliaga) [inline modules/110-istio/docs/README.md:253]:
```suggestion
* The cluster domains in the [`clusterDomain`](../../installing/configuration.html#clusterconfiguration-clusterdomain) parameter of the _ClusterConfiguration_ interface must be the same for all multi-cluster members, the default value is `cluster.local`.
```

File: modules/110-istio/docs/README.md [hunk 11]
```diff
@@ -243,6 +248,11 @@ To establish a federation, you must:
...
+* Pod and Service subnets **should** be unique for each multicluster member ([podSubnetCIDR](../../installing/configuration.html#clusterconfiguration-podsubnetcidr) and [serviceSubnetCIDR](../../installing/configuration.html#clusterconfiguration-servicesubnetcidr) settings in `ClusterConfiguration` resource).
```
Comment (elenashliaga) [inline modules/110-istio/docs/README.md:254]:
```suggestion
* Pod and Service subnets in the [`podSubnetCIDR`](../../installing/configuration.html#clusterconfiguration-podsubnetcidr) and [`serviceSubnetCIDR`](../../installing/configuration.html#clusterconfiguration-servicesubnetcidr) parameters of the _ClusterConfiguration_ interface must be unique for each multicluster member.
```

File: modules/110-istio/docs/README_RU.md (hunk 3 above)
Comment (z9r5) [reply modules/110-istio/docs/README_RU.md:220]: @apolovov Это ли не ложь?

File: modules/110-istio/docs/README_RU.md [hunk 12]
```diff
@@ -246,6 +251,11 @@ Istio позволяет осуществлять сбор трейсов с п
 
 ### Мультикластер
 
+#### Требования к кластерам
+
+* Домены кластеров в параметре [`clusterDomain`](../../installing/configuration.html#clusterconfiguration-clusterdomain) ресурса [_ClusterConfiguration_](../../installing/configuration.html#clusterconfiguration) должны быть одинаковыми для всех членов мультикластера, по умолчанию значение параметра `cluster.local`.
```
Comment (z9r5) [inline modules/110-istio/docs/README_RU.md:256]:
```suggestion
* Домены кластеров в параметре [`clusterDomain`](../../installing/configuration.html#clusterconfiguration-clusterdomain) ресурса [_ClusterConfiguration_](../../installing/configuration.html#clusterconfiguration) должны быть одинаковыми для всех членов мультикластера. По умолчанию, значение параметра `cluster.local`.
```
@elenashliaga

File: modules/110-istio/docs/README.md [hunk 13]
```diff
@@ -243,6 +248,11 @@ To establish a federation, you must:
 
 ### Multicluster
 
+#### Requirements for clusters
+
+* Cluster domains in the [`clusterDomain`](../../installing/configuration.html#clusterconfiguration-clusterdomain) parameter of the resource [_ClusterConfiguration_](../../installing/configuration.html#clusterconfiguration) must be the same for all multicluster members, the default value is `cluster.local`.
```
Comment (z9r5) [inline modules/110-istio/docs/README.md:253]:
```suggestion
* Cluster domains in the [`clusterDomain`](../../installing/configuration.html#clusterconfiguration-clusterdomain) parameter of the resource [_ClusterConfiguration_](../../installing/configuration.html#clusterconfiguration) must be the same for all multicluster members. The default value is `cluster.local`.
```

File: modules/110-istio/docs/README.md [hunk 14]
```diff
@@ -210,6 +210,11 @@ Below are their fundamental differences:
 
 ### Federation
 
+#### Requirements for clusters
+
+* Each cluster must have a unique domain in the [`clusterDomain`](../../installing/configuration.html#clusterconfiguration-clusterdomain) parameter of the resource [_ClusterConfiguration_](../../installing/configuration.html#clusterconfiguration), the default value is `cluster.local`.
```
Comment (z9r5) [inline modules/110-istio/docs/README.md:215]:
```suggestion
* Each cluster must have a unique domain in the [`clusterDomain`](../../installing/configuration.html#clusterconfiguration-clusterdomain) parameter of the resource [_ClusterConfiguration_](../../installing/configuration.html#clusterconfiguration). The default value is `cluster.local`.
```

**Review by z9r5 — APPROVED** [review]

**Comment by deckhouse-BOaTswain** [comment]
Cherry pick PR [7881](https://github.com/deckhouse/deckhouse/pull/7881) to the branch [release-1.58](https://github.com/deckhouse/deckhouse/tree/release-1.58) successful!

**Pull request merged**